healthcheck/utils.py

```diff
@@ -0,0 +1,26 @@
+import errno
+import os
+
+
+def file_exists(path):
+    """Return True if a file exists at `path` (even if it can't be read),
+    otherwise False.
+
+    This is different from os.path.isfile and os.path.exists which return
+    False if a file exists but the user doesn't have permission to read it.
```

Comment:
```Python
>>> os.stat("/tmp/foo/bar")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OSError: [Errno 13] Permission denied: '/tmp/foo/bar'
```

Which is different to the file not existing. In particular it tells you that it's not possible to know whether or not the file exists.